Docs/ServletSingleSignOnSecurityFilter.md

```diff
@@ -32,6 +32,7 @@ Filter Options
 
 The filter can be configured with the following `init-param` options. 
 
+* disableSSO: Allows for the enabling/disabling of Single-SignOn Security Filter.This is useful if you require to bypass the SSO process for troubleshooting/diagnostics and provide login by some other means.
```

Comment:
Feels to me that you never would want this feature and if you need to disable SSO for troubleshooting or diagnostics you would comment out/delete the filter. 

This is also why the spec for these filters doesn't include an `enabled` feature, it can be accomplished as easily bu other means.

Reply:
I can imagine this need for a newly onboarded team member that needs to debug and code within the app but is waiting on getting proper AD access to the app.  Sure option is to disable filter by other means but this is nicer I think.  Plus some automation can be put around the process, such as a bash script that would ask the user if they wanted it on or off.

Reply:
We have an on-prem offering of our solution where our client's have the ability to enable or disable the use of SSO, and we automate the restarting of the server with whatever selection they've made. Makes it much easier than having a client find and go through xml files to comment/remove xml fragments.

Reply:
That makes sense, so my comment is really only about naming of this option and the location where it's implemented.